src/styles/theme.scss

```diff
@@ -337,22 +411,40 @@ h5 {
   background-color: $primary-colour;
 }
 
+.card--alternate.card--with-icon {
+  background-color: transparent;
+  color: $secondary-colour;
+  border: 2px solid $icon-background;
+  border-radius: 10px;
+  article h2 {
```

Comment:
We are also trying not to nest here. Is the text the wrong colour for another reason?

Reply:
text and h2 are the same colour if I don't change it here. I will not nest but then I thought that's what sass/scss is - among others - allowing us to do, i.e. less code.

Reply:
It is what sass lets people do. Matt deliberately un-sassified this project into single use classes. We left it in since it's [art of the build but are trying to keep as much to that style as possible.

Reply:
There's an argument that sass/ less code is bad since it obscures things for human readers. Harder to search for stuff and figure out what is acting on what. I'm sure there is reading around it if you are interested and have free space in your life. :)